cmake/scripts/generate_config.cmake

```diff
@@ -19,7 +20,7 @@ set(DAL_ROOT_REL_PATH "../../..")
 set(INC_REL_PATH "include")
 set(LIB_REL_PATH "lib")
 set(DLL_REL_PATH "redist")
-set(SUB_DIR "intel64")
+set(ARCH_DIR "${ARCH_DIR}")
```

Comment:
Let's use ARCH_DIR_ONEDAL here as well (and in makefile and oneDALConfig.cmake.in)

Reply:
Thanks for the review. We have incorporated this comment in the latest commit pushed. Thanks